.github/dependabot.yml

```
@@ -9,3 +10,24 @@ updates:
     labels:
       - area/dependencies
       - area/ci
+  # keycloak UI dependencies
+  - package-ecosystem: npm
+    directory: /themes/src/main/resources/theme/keycloak/common/resources/
+    schedule:
+      interval: daily
+      time: "00:01"
+      timezone: UTC
+    open-pull-requests-limit: 6
```

I'd like to see all updates at once, so let's change this to `999`.
```suggestion
    open-pull-requests-limit: 999
```